docs/src/Video.doc.js

```diff
@@ -88,6 +88,12 @@ card(
         description: `This \`children\` prop is not same as children inside the native html \`video\` element.
           Instead it serves to add overlays on top of the html video element, while still being under the video controls.`,
       },
+      {
+        name: 'crossOrigin',
+        type: 'string',
+        description:
+          "Designate CORS behavior for the video element, one of 'use-credentials' or 'anonymous'. Note: when not passed in, CORS checks are disabled.",
+      },
```

Comment:
type: 'use-credentials' | 'anonymous',

please make it explicit to the 2 strigs.

Reply:
description: "Designate CORS behavior for the video element. When not passed in, CORS checks are disabled."